_rules/aria-hidden-focus-6cfa84.md

```diff
@@ -46,10 +46,10 @@ By adding `aria-hidden="true"` to an element, content authors ensure that assist
 
 A focusable element with `aria-hidden="true"` is ignored as part of the reading order, but still part of the focus order, making it's state of visible or hidden unclear.
 
-- https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
-- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
-- https://www.w3.org/TR/wai-aria-1.1/#aria-hidden
-- https://www.w3.org/TR/html/editing.html#can-be-focused
+- [Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
+- [Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)
+- [`aria-hidden` (state)](https://www.w3.org/TR/wai-aria-1.1/#aria-hidden)
+- [HTML Living Standard - Focusable areas](https://html.spec.whatwg.org/multipage/interaction.html#focusable-area)
```

Comment:
Not sure this is the correct section. @kasperisager, can you tell? I will also try to look further into it.

Reply:
The old one was this: https://html.spec.whatwg.org/multipage/interaction.html#specially-focusable

Reply:
Looking at the rest of the rule, only the "sequential focus navigation" term is used, so I don't think we actually need to link to another term that has to do with focus in the background section of this rule. Seems out of place to me. 

However, the link used for "sequential focus navigation" in this rule is also broken. It should be changed to: https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation.

For the review of this rule I only looked at the links changed - which on further inspection seems to only be in the background sections. 
What about the links used in the prose of the applicability, expectations, assumptions etc.?
What is the plan for checking them for WCAG 2.0 references and broken links to the old HTML spec?